src/Client/Connection.h

```diff
@@ -227,7 +227,7 @@ class Connection : public IServerConnection
     UInt64 server_version_patch = 0;
     UInt64 server_revision = 0;
     UInt64 server_parallel_replicas_protocol_version = 0;
-    UInt64 server_cluster_function_protocol_version = DBMS_CLUSTER_INITIAL_PROCESSING_PROTOCOL_VERSION;
```

Comment:
This is changing the value:

https://github.com/ClickHouse/ClickHouse/blob/5d41e7d7d60e28e50d10b2055505565205725331/src/Core/ProtocolDefines.h#L36

It used to  be 1